assets/js/modules/tagmanager/datastore/containers.js

```diff
@@ -404,6 +404,39 @@ const baseSelectors = {
 	isDoingCreateContainer( state ) {
 		return Object.values( state.isFetchingCreateContainer ).some( Boolean );
 	},
+	/**
+	 * Gets all containers for all accounts.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return {(Array|undefined)} Array of containers, or `undefined` if not loaded yet.
+	 */
+	getAllContainers: createRegistrySelector( ( select ) => () => {
```

Comment:
I believe this was added to be able to find the account (and container) in the presence of an existing tag which was previously done on the server side as part of the permission check. Going forward we won't be auto-selecting values for a detected existing tag (the inputs will be controlled by the saved value only) so this won't be necessary.

There are a few other things that aren't quite right in this selector but I'll omit those details as we won't be keeping it.

Reply:
Thanks for spotting this @aaemnnosttv, I must confess that I didn't spot @kuasha420's recent commit. I had misread this paragraph of Arafat's [comment](https://github.com/google/site-kit-wp/pull/4750#issuecomment-1046185340).

> For not auto selecting correct container, I had to loop over all the accounts and try to find the account that has the container that's present in the existing tag, and if found, we can select the account and container. I am not sure if that's the best approach here.

I mistakenly read that as a question and the rest of the comment as essentially saying that we don't need to do this now as it will be addressed in #4713. I therefore approved the PR on the basis of my previously review and testing, thinking nothing had been added.